src/components/application_manager/src/message.cc

```diff
@@ -98,11 +107,13 @@ bool Message::operator==(const Message& message) {
   bool version = version_ == message.version_;
   bool data_size = data_size_ == message.data_size_;
   bool payload_size = payload_size_ == message.payload_size_;
-
-  bool binary_data = std::equal(binary_data_->begin(),
-                                binary_data_->end(),
-                                message.binary_data_->begin(),
-                                BinaryDataPredicate);
+  bool binary_data = true;
```

Comment:
@JackLivio it fixed core crash. Variable probably should be renamed  to binary_data_equals